jakttest/process.cpp

```diff
@@ -92,7 +92,7 @@ ErrorOr<Optional<i32>> poll_process_exit(i32 pid)
     }
     // NOTE: compiler seems to struggle to find a constructor when
     // nesting error & optional
-    return Optional<i32>(exit_code);
+    return Optional<Tuple<bool, i32>>(Tuple { WIFEXITED(wstatus) != 0, WEXITSTATUS(wstatus) });
```

Comment:
Return a real struct with named members instead of (ab)using Tupe?

Reply:
I can't think of a name for such a structure. Can you suggest one?

Reply:
Nevermind, I could use a `RawExitStatus` struct that contains this info and can be used by Jakt code.